### PR TITLE
NN-4500 all local unit/int tests pass

### PR DESCRIPTION
**Pull request by HamHazJustice**
Check your answers - added Date / Time of discovery

### CODE REVIEW

File: server/services/placeOnReportService.ts [hunk 1]
```diff
@@ -156,17 +159,25 @@ export default class PlaceOnReportService {
         value: getFormattedOfficerName(reporter.name),
       },
       {
-        label: 'Date',
+        label: 'Date of Incident',
```
Comment (timoj58) [inline server/services/placeOnReportService.ts:162]: incident should be lower looking at the wireframe.  Design wise i think Anne always prefers upper case on first word of sentence only

File: server/services/placeOnReportService.ts [hunk 2]
```diff
@@ -156,17 +159,25 @@ export default class PlaceOnReportService {
...
         value: date,
       },
       {
-        label: 'Time',
+        label: 'Time of Incident',
```
Comment (timoj58) [inline server/services/placeOnReportService.ts:166]: ditto here lower case incident

File: server/services/placeOnReportService.ts (hunk 1 above)
Comment (natclamp-moj) [reply server/services/placeOnReportService.ts:162]: Yep Tim is correct

Comment (HamHazJustice) [reply server/services/placeOnReportService.ts]: addressed throughout

File: server/services/placeOnReportService.ts (hunk 2 above)
Comment (HamHazJustice) [reply server/services/placeOnReportService.ts]: addressed throughout

**Review by natclamp-moj — APPROVED** [review]
Looks good

**Pull request merged**